src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4041,6 +4045,36 @@ private static bool IsNegativeConstantForArraySize(BoundExpression expression)
             {
                 analyzedArguments.Free();
             }
+
+            static void validateRecordCopyConstructor(MethodSymbol constructor, NamedTypeSymbol baseType, MethodSymbol resultMember, Location errorLocation, DiagnosticBag diagnostics)
+            {
+                if (IsUserDefinedRecordCopyConstructor(constructor))
+                {
+                    if (baseType.SpecialType == SpecialType.System_Object)
+                    {
+                        if (resultMember.ContainingType.SpecialType != SpecialType.System_Object)
```

Comment:
>resultMember [](start = 28, length = 12)

Can this be null?

Reply:
Thought about it then zoned out. Thanks for catching

---
In reply to: [439107744](https://github.com/dotnet/roslyn/pull/45022#discussion_r439107744) [](ancestors = 439107744)